react/src/pages/dashboard/ParticipantTable.tsx

```diff
@@ -77,14 +79,14 @@ export default function ParticipantTable() {
                 />}
            <MaterialTable
                 columns={[
-                    { title: 'Participant Codename', field: 'participant_codename', align: 'center'},
-                    { title: 'Family Codename', field: 'family_codename', align: 'center'},
-                    { title: 'Participant Type', field: 'participant_type' , align: 'center', lookup: participantTypes, defaultFilter: filter},
-                    { title: 'Affected', field: 'affected', type: 'boolean', align: 'center'},
-                    { title: 'Solved', field: 'solved', type: 'boolean', align: 'center'},
-                    { title: 'Sex', field: 'sex', type: 'string', align: 'center', lookup: sexTypes},
+                    { title: 'Participant Codename', field: 'participant_codename', align: 'center' },
+                    { title: 'Family Codename', field: 'family_codename', align: 'center' },
```

Comment:
This field is not supposed to be editable for now.

Reply:
The codename fields?

Reply:
Only family codename.

Reply:
👌